config/graph.mainnet.yml

```diff
@@ -58,6 +58,8 @@ contracts:
     calls:
       - fn: "addMinter"
         minter: "${{RewardsManager.address}}"
+      - fn: "addMinter"
+        minter: "${{L1GraphTokenGateway.address}}"
```

Comment:
We need to add this to the goerli and localhost config no?

Reply:
Good point, will add it